app/models/post.rb

```diff
@@ -3,6 +3,8 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
+
```

Comment:
- [ ]  Kindly, you should also validate the `comments_counter`, and the `like_counter` as the requirements say that they must be integers greater than or equal to zero.
Example :point_down: 

```rb
  validates_numericality_of :comments_counter, only_integer: true, greater_than_or_equal: 0
  validates_numericality_of :like_counter, only_integer: true, greater_than_or_equal: 0
```